proxy/hdrs/HTTP.cc

```diff
@@ -2130,6 +2130,87 @@ HTTPInfo::unmarshal(char *buf, int len, RefCountObj *block_ref)
   return alt->m_unmarshal_len;
 }
 
+int
+HTTPInfo::unmarshal_v24_1(char *buf, int len, RefCountObj *block_ref)
+{
```

Comment:
Also I can merge it into `HTTPInfo::unmarshal` but not sure whether it is deserved.